equinox-shipment/Domain/Shipment.fs

```diff
@@ -0,0 +1,67 @@
+﻿module Shipment
+
+open System
+
+type Shipment = { id: string; containerId: string }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | ShipmentCreated    of shipmentId  : string
```

Comment:
tracking the id is not useful so best to elide here

Reply:
Removed thanks!